electron/main.js

```diff
@@ -0,0 +1,40 @@
+import { join } from "path";
```

Comment:
I cleaned up this file a bit:
- Removed most of the boilerplate comments
- Removed debugging `console.log`'s, the ones that remain were intentionally left and should only be visible in `dev` mode.